src/routers/upload.py

```diff
@@ -81,82 +208,98 @@ async def upload_geotiff(file: UploadFile, token: Annotated[str, Depends(oauth2_
-    with target_path.open('rb') as f:
+    with target_path.open("rb") as f:
         sha256 = hashlib.sha256(f.read()).hexdigest()
 
     # try to open with rasterio
-    with rasterio.open(str(target_path), 'r') as src:
+    with rasterio.open(str(target_path), "r") as src:
         bounds = src.bounds
-        transformed_bounds = rasterio.warp.transform_bounds(src.crs, 'EPSG:4326', *bounds)
-    
+        transformed_bounds = rasterio.warp.transform_bounds(
+            src.crs, "EPSG:4326", *bounds
+        )
+
     # stop the timer
     t2 = time.time()
 
     # fill the metadata
-    #dataset = Dataset(
+    # dataset = Dataset(
     data = dict(
         file_name=target_path.name,
         file_alias=file.filename,
         file_size=target_path.stat().st_size,
         copy_time=t2 - t1,
         sha256=sha256,
-        #bbox=f"BOX({bounds.bottom} {bounds.left}, {bounds.top} {bounds.right})",
+        # bbox=f"BOX({bounds.bottom} {bounds.left}, {bounds.top} {bounds.right})",
         bbox=transformed_bounds,
         status=StatusEnum.pending,
-        user_id=user.id
+        user_id=user.id,
     )
     # print(data)
-    dataset=Dataset(**data)
+    dataset = Dataset(**data)
 
     # upload the dataset
     with use_client(token) as client:
         try:
-            send_data = {k: v for k, v in dataset.model_dump().items() if k != 'id' and v is not None}
+            send_data = {
+                k: v
+                for k, v in dataset.model_dump().items()
+                if k != "id" and v is not None
+            }
             response = client.table(settings.datasets_table).insert(send_data).execute()
         except Exception as e:
-            logger.exception(f"An error occurred while trying to upload the dataset: {str(e)}", extra={"token": token, "user_id": user.id})
-            raise HTTPException(status_code=400, detail=f"An error occurred while trying to upload the dataset: {str(e)}")
-    
+            logger.exception(
+                f"An error occurred while trying to upload the dataset: {str(e)}",
+                extra={"token": token, "user_id": user.id},
+            )
+            raise HTTPException(
+                status_code=400,
+                detail=f"An error occurred while trying to upload the dataset: {str(e)}",
+            )
+
     # update the dataset with the id
     dataset = Dataset(**response.data[0])
-    
+
     # do some monitoring
     monitoring.uploads_counter.inc()
     monitoring.upload_time.observe(dataset.copy_time)
     monitoring.upload_size.observe(dataset.file_size)
 
     logger.info(
-        f"Created new dataset <ID={dataset.id}> with file {dataset.file_alias}. ({format_size(dataset.file_size)}). Took {dataset.copy_time:.2f}s.", 
-        extra={"token": token, "user_id": user.id, "dataset_id": dataset.id}
+        f"Created new dataset <ID={dataset.id}> with file {dataset.file_alias}. ({format_size(dataset.file_size)}). Took {dataset.copy_time:.2f}s.",
```

Comment:
Should we also print the original dataset name here? Not just the alias?

Reply:
the file_alias is the original file name :) But this is not obvious.